source/config.hpp

```diff
@@ -90,6 +91,7 @@ class Config
 	string const &prefix_for_static_member_functions() { return prefix_for_static_member_functions_; }
 
 	string const &holder_type() const { return holder_type_; }
+	string const &include_file() const { return include_file_; }
```

Comment:
same as above: could you please rename this to `pybind11_include_file` to avoid confusion? Thanks,